node-tests/acceptance/exam-test.js

```diff
@@ -11,7 +11,7 @@ function getNumberOfTests(str) {
   return match && parseInt(match[1], 10);
 }
 
-var TOTAL_NUM_TESTS = 48;
+var TOTAL_NUM_TESTS = 50;
```

Comment:
this change came from two new files added by ember-cli blueprint:
`tests/dummy/app/resolver.js` and `tests/helpers/module-for-acceptance.js`